doc/user/content/guides/dbt.md

```diff
@@ -0,0 +1,276 @@
+
+### dbt models
+
+An important difference in using dbt with Materialize is that **your models stay up-to-date** without the need for manual or configured refreshes. This allows you to efficiently transform streaming data using the same mental model you'd use for batch transformations on top of any other database.
+
+1. Create a model for each SQL statement you're planning to deploy. Each individual model should be stored as a `.sql` file under the directory defined by `source-paths` in `dbt_project.yml`.
+
+    As an example, we'll use the SQL statements in our [getting started guide](/get-started/) and re-write them as dbt models.
+
+    <h5>Creating a source</h5>
+
+    You can instruct dbt to create a [source](/sql/create-source) in Materialize using the custom `source` [materialization](#materializations):
+
+    ```sql
+    {{ config(materialized='source') }}
+
+    {% set source_name %}
+        {{ mz_generate_name('market_orders_raw') }}
+    {% endset %}
+
+    CREATE SOURCE {{ source_name }}
+    FROM PUBNUB
+    SUBSCRIBE KEY 'sub-c-4377ab04-f100-11e3-bffd-02ee2ddab7fe'
+    CHANNEL 'pubnub-market-orders'
+    ```
+
+    The `mz_generate_name` [macro](https://docs.getdbt.com/docs/building-a-dbt-project/jinja-macros/#macros) allows you to generate a fully-qualified name from a base object name. Here, `source_name` would be compiled to `materialize.public.market_orders_raw`.
+
+    <h5>Creating a view</h5>
+
+    dbt models are materialized as `views` by default, so to create a [view](/sql/create-view) in Materialize you can simply provide the SQL statement in the model (and skip the `materialized` configuration parameter):
+
+    ```sql
+    SELECT
+        ((text::jsonb)->>'bid_price')::float AS bid_price,
+        (text::jsonb)->>'order_quantity' AS order_quantity,
+        (text::jsonb)->>'symbol' AS symbol,
+        (text::jsonb)->>'trade_type' AS trade_type,
+        to_timestamp(((text::jsonb)->'timestamp')::bigint) AS ts
+    FROM {{ source('public','market_orders_raw') }}
+    ```
+
+    One thing to note here is that the model depends on the source defined in the previous step. To express this dependency and track the **lineage** of your project, you can use the dbt [source()](https://docs.getdbt.com/docs/building-a-dbt-project/using-sources) and [ref()](https://docs.getdbt.com/reference/dbt-jinja-functions/ref) functions.
+
+    <h5>Creating a materialized view</h5>
+
+    This is where Materialize goes beyond dbt's incremental models (and traditional databases), with [materialized views](/sql/create-materialized-view) that **continually update** as the underlying data changes:
+
```

Comment:
Should this be "incrementally" instead of "continually"? I thought any database's materialized view is updated continually, it's just that it's usually recomputed from scratch and takes a relatively long time to do.  But I may be misunderstanding this.

Reply:
Wanted to avoid "incrementally" because there's a mention to dbt's "incremental models" immediately before. But the correct word is "continuously", not "continually".